*** tests/A-01-04-M-ok.out	2020-10-30 22:42:03.000000000 +0000
--- tests/A-01-04-M-ok.outhyp	2020-11-09 13:04:06.799708400 +0000
***************
*** 43,49 ****
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
--- 43,50 ----
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Guardar: Operação inválida: Problema ao abrir 'null'.
! Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
