*** tests/A-01-03-M-ok.out	2020-10-30 22:41:57.000000000 +0000
--- tests/A-01-03-M-ok.outhyp	2020-11-09 13:04:06.413781000 +0000
***************
*** 29,41 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: E4|Filomena Figueiredo|Lisboa, Portugal|NORMAL|0|0
! ER|Abdul Figueiredo|Casablanca, Morocco|NORMAL|0|0
! H2O|Rohit Figueiredo|New Delhi, India|NORMAL|0|0
! H2SO4|John Figueiredo|Wellington, New Zealand|NORMAL|0|0
! O9|Hellen Figueiredo|San Francisco, CA, USA|NORMAL|0|0
! R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0
! Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
  3 - Registar cliente
--- 29,35 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
  3 - Registar cliente
***************
*** 60,69 ****
  3 - Permitir/inibir transacções
  4 - Consultar transacções
  0 - Sair
! Escolha uma opção: S1|First one|Lisboa|SIM
! S2|Second one|Lisbon|SIM
! S3|Third one|Lisbonne|SIM
! Gestão de Fornecedores
  1 - Mostrar fornecedores
  2 - Registar
  3 - Permitir/inibir transacções
--- 54,60 ----
  3 - Permitir/inibir transacções
  4 - Consultar transacções
  0 - Sair
! Escolha uma opção: Gestão de Fornecedores
  1 - Mostrar fornecedores
  2 - Registar
  3 - Permitir/inibir transacções
