*** tests/A-17-02-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-17-02-M-ok.outhyp	2020-12-03 16:57:09.798569000 +0000
***************
*** 16,28 ****
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador do fornecedor: Identificador do produto: Quantidade: Encomendar mais produtos? (s/n) Identificador do produto: Quantidade: Encomendar mais produtos? (s/n) Registar encomenda: Operação inválida: O produto 'qwewe' não existe.
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
--- 16,29 ----
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador do fornecedor: Identificador do fornecedor: Identificador do produto: Quantidade: Quantidade: Quantidade: Registar encomenda: Operação inválida: O produto '2' não existe.
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
