*** tests/A-18-02-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-18-02-M-ok.outhyp	2020-12-04 11:43:17.045005000 +0000
***************
*** 23,29 ****
  4 - Pagar
  0 - Sair
  Escolha uma opção: Identificador da transacção: 0|P1|20|0
- 1C6H5OH|10
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
--- 23,28 ----
***************
*** 37,43 ****
  4 - Pagar
  0 - Sair
  Escolha uma opção: Identificador da transacção: 0|P1|20|0
- 1C6H5OH|10
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
--- 36,41 ----
