*** tests/A-14-02-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-14-02-M-ok.outhyp	2020-12-03 15:27:38.777402600 +0000
***************
*** 17,30 ****
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: Identificador do produto: Titulo do livro: Autor do livro: ISBN: Preço: Valor crítico para as existências: Identificador do fornecedor: Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
  3 - Registar Contentor
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: BOOK|BOOK1|P1|22|12|0|título|autor|isbn
  Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
--- 17,30 ----
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: Identificador do produto: Autor do livro: Titulo do livro: ISBN: Preço: Valor crítico para as existências: Identificador do fornecedor: Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
  3 - Registar Contentor
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: BOOK|BOOK1|P1|22|12|0|autor|título|isbn
  Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
