*** tests/A-14-01-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-14-01-M-ok.outhyp	2020-12-03 15:27:38.480662100 +0000
***************
*** 24,30 ****
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: Identificador do produto: Preço: Valor crítico para as existências: Identificador do fornecedor: Tipo de serviço: Qualidade de serviço: Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
  3 - Registar Contentor
--- 24,30 ----
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: Identificador do produto: Preço: Valor crítico para as existências: Identificador do fornecedor: Tipo de serviço: Tipo de serviço: Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
  3 - Registar Contentor
