*** tests/A-09-06-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-09-06-M-ok.outhyp	2020-12-03 15:05:23.293038900 +0000
***************
*** 30,39 ****
  0 - Sair
  Escolha uma opção: P1|Publicações Europa-América|Lisboa, Portugal|SIM
  P3|O’Reilly|Köln, Germany|SIM
- qq|Mais UM|Não sei.|SIM
  S1|Toshiba|Tokyo, Japan|SIM
- sSs|outro|outra vez|SIM
  W2|Papelaria Fernandes|Oeiras, Portugal|SIM
  Gestão de Fornecedores
  1 - Mostrar fornecedores
  2 - Registar
--- 30,39 ----
  0 - Sair
  Escolha uma opção: P1|Publicações Europa-América|Lisboa, Portugal|SIM
  P3|O’Reilly|Köln, Germany|SIM
  S1|Toshiba|Tokyo, Japan|SIM
  W2|Papelaria Fernandes|Oeiras, Portugal|SIM
+ qq|Mais UM|Não sei.|SIM
+ sSs|outro|outra vez|SIM
  Gestão de Fornecedores
  1 - Mostrar fornecedores
  2 - Registar
