*** tests/A-18-04-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-18-04-M-ok.outhyp	2020-12-03 15:05:38.777868300 +0000
***************
*** 22,28 ****
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: 0|R2|C|1|300|300|4
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
--- 22,29 ----
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: CONTAINER|C|P2|300|20|100|AIR|C4
! CONTAINER|C|P2|300|20|100|AIR|C40|R2|C|1|300|0.0|4|NÃO|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
***************
*** 35,41 ****
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: 0|R2|C|1|300|300|4|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
--- 36,43 ----
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: CONTAINER|C|P2|300|20|100|AIR|C4
! CONTAINER|C|P2|300|20|100|AIR|C40|R2|C|1|300|0.0|4|NÃO|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
***************
*** 61,67 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|SELECTION|300|300
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 63,69 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
