*** tests/A-03-03-M-ok.out	2020-10-30 22:43:02.000000000 +0000
--- tests/A-03-03-M-ok.outhyp	2020-11-08 16:48:32.807191500 +0000
***************
*** 17,23 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: H2O|Rohit Figueiredo|New Delhi, India|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 17,23 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: Mostrar cliente: Operação inválida: O cliente 'H2O' não existe.
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
***************
*** 25,31 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: E4|Filomena Figueiredo|Lisboa, Portugal|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 25,31 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: Mostrar cliente: Operação inválida: O cliente 'E4' não existe.
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
***************
*** 33,39 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 33,39 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: Mostrar cliente: Operação inválida: O cliente 'R2' não existe.
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
