*** tests/A-14-05-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-14-05-M-ok.outhyp	2020-12-03 16:50:47.136785600 +0000
***************
*** 35,48 ****
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: Identificador do produto: Titulo do livro: Autor do livro: ISBN: Preço: Valor crítico para as existências: Identificador do fornecedor: Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
  3 - Registar Contentor
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: Identificador do produto: Preço: Valor crítico para as existências: Identificador do fornecedor: Tipo de serviço: Qualidade de serviço: Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
  3 - Registar Contentor
--- 35,48 ----
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: Identificador do produto: Autor do livro: Titulo do livro: ISBN: Preço: Valor crítico para as existências: Identificador do fornecedor: Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
  3 - Registar Contentor
  4 - Registar Livro
  5 - Alterar Preço
  0 - Sair
! Escolha uma opção: Identificador do produto: Preço: Valor crítico para as existências: Identificador do fornecedor: Tipo de serviço: Tipo de serviço: Gestão de Produtos
  1 - Visualizar
  2 - Registar Caixa
  3 - Registar Contentor
***************
*** 51,57 ****
  0 - Sair
  Escolha uma opção: BOX|189|P1|1|8|0|PERSONAL
  BOX|1C6H5OH|P1|2|20|100|NORMAL
! BOOK|334|P2|1|4|0|tt|aa|qwrwerw
  BOX|4CO2|P2|16|20|0|PERSONAL
  CONTAINER|7M5|P2|8|20|100|EXPRESS|B4
  CONTAINER|9999|P2|9|9|0|NORMAL|C4
--- 51,57 ----
  0 - Sair
  Escolha uma opção: BOX|189|P1|1|8|0|PERSONAL
  BOX|1C6H5OH|P1|2|20|100|NORMAL
! BOOK|334|P2|1|4|0|aa|tt|qwrwerw
  BOX|4CO2|P2|16|20|0|PERSONAL
  CONTAINER|7M5|P2|8|20|100|EXPRESS|B4
  CONTAINER|9999|P2|9|9|0|NORMAL|C4
