*** tests/A-17-04-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-17-04-M-ok.outhyp	2020-12-04 12:45:55.229209100 +0000
***************
*** 23,29 ****
  4 - Pagar
  0 - Sair
  Escolha uma opção: Identificador da transacção: 0|P2|1616|0
- 3O3|202
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
--- 23,28 ----
***************
*** 49,55 ****
  4 - Consultar transacções
  0 - Sair
  Escolha uma opção: Identificador do fornecedor: 0|P2|1616|0
- 3O3|202
  Gestão de Fornecedores
  1 - Mostrar fornecedores
  2 - Registar
--- 48,53 ----
