*** tests/A-18-12-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-18-12-M-ok.outhyp	2020-12-05 18:25:48.792072500 +0000
***************
*** 170,176 ****
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
--- 170,177 ----
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Guardar: Operação inválida: Problema ao abrir 'null'.
! Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
