*** tests/A-18-06-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-18-06-M-ok.outhyp	2020-12-04 12:45:58.322412200 +0000
***************
*** 54,60 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: R2|Jorge Figueiredo|Lisboa, Portugal|ELITE|2800|2800
  S2|Ricardo Figueiredo|Lisboa|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
--- 54,60 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|2800|0
  S2|Ricardo Figueiredo|Lisboa|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
